app/assets/javascripts/hydra-editor/manage_repeating_fields.js

```diff
@@ -8,8 +8,8 @@ var HydraEditor = (function($) {
           this.options = options;
 
           this.controls = $("<span class=\"input-group-btn field-controls\">");
-          this.remover  = $("<button class=\"btn btn-danger remove\"><i class=\"icon-white glyphicon-minus\"></i><span>Remove</span></button>");
-          this.adder    = $("<button class=\"btn btn-success add\"><i class=\"icon-white glyphicon-plus\"></i><span>Add</span></button>");
+          this.remover  = $("<button type=\"button\" class=\"btn btn-danger remove\"><i class=\"icon-white glyphicon-minus\"></i><span>Remove</span></button>");
```

Comment:
Also on the page @mtribone mentioned:
> The type of the button. Possible values are:  submit: The button submits the form data to the server. This is the default if the attribute is not specified, or if the attribute is dynamically changed to an empty or invalid value.

So we probably should set it to `type=button` :+1: